File: noNonsense/01_two_pointers/skeleton.java
public class skeleton {

    public static void main(String Args) {

        int[] arr = {1, 2, 3, 4, 5, 9};
        int target = 12 ;
        int i = 0 ,j = arr.length - 1 ;
        while(i<j){
            int sum = arr[i] + arr[j];
            if (sum == target){
                System.out.println(i + " " + j );
                break;
            }
            else if (sum < target) {
                i++
            }
            else
        }


    }
    
}

File: noNonsense/01_two_pointers/skeleton.py
arr = [1,2,3,4,5,6,7,8,9,10]
target = 12 

l = 0 
r = len(arr) - 1

while l < r :
    sum = arr[l] + arr[r]

    if sum == target :
        print([l,r])
        l += 1 
        r -= 1 
    elif sum < target :
        l += 1
    else :
        r -= 1 

